manifests/integreatly-amq-online/amq-online-1.4.1/amq-online.1.4.1.clusterserviceversion.yaml

```diff
@@ -630,65 +630,65 @@ spec:
-                  value: registry.redhat.io/amq7/amq-online-1-standard-controller:1.4
+                  value: registry.redhat.io/amq7/amq-online-1-standard-controller:1.4-54.1587495424
                 - name: AGENT_IMAGE
-                  value: registry.redhat.io/amq7/amq-online-1-agent:1.4
+                  value: registry.redhat.io/amq7/amq-online-1-agent:1.4-57
                 - name: BROKER_IMAGE
-                  value: registry.redhat.io/amq7/amq-broker:7.6
+                  value: registry.redhat.io/amq7/amq-broker:7.6-5
                 - name: BROKER_PLUGIN_IMAGE
-                  value: registry.redhat.io/amq7/amq-online-1-broker-plugin:1.4
+                  value: registry.redhat.io/amq7/amq-online-1-broker-plugin:1.4-56.1587495474
                 - name: TOPIC_FORWARDER_IMAGE
-                  value: registry.redhat.io/amq7/amq-online-1-topic-forwarder:1.4
+                  value: registry.redhat.io/amq7/amq-online-1-topic-forwarder:1.4-57.1587495421
                 - name: MQTT_GATEWAY_IMAGE
-                  value: registry.redhat.io/amq7/amq-online-1-mqtt-gateway:1.4
+                  value: registry.redhat.io/amq7/amq-online-1-mqtt-gateway:1.4-56.1587495439
                 - name: MQTT_LWT_IMAGE
-                  value: registry.redhat.io/amq7/amq-online-1-mqtt-lwt:1.4
+                  value: registry.redhat.io/amq7/amq-online-1-mqtt-lwt:1.4-57.1587495436
                 - name: NONE_AUTHSERVICE_IMAGE
-                  value: registry.redhat.io/amq7/amq-online-1-none-auth-service:1.4
+                  value: registry.redhat.io/amq7/amq-online-1-none-auth-service:1.4-56.1587495432
                 - name: KEYCLOAK_IMAGE
                   value: registry.redhat.io/redhat-sso-7/sso73-openshift:latest
                 - name: KEYCLOAK_PLUGIN_IMAGE
-                  value: registry.redhat.io/amq7/amq-online-1-auth-plugin:1.4
+                  value: registry.redhat.io/amq7/amq-online-1-auth-plugin:1.4-55.1587495477
                 - name: CONTROLLER_ENABLE_CONSOLE_SERVICE
                   value: "true"
                 - name: CONSOLE_INIT_IMAGE
-                  value: "registry.redhat.io/amq7/amq-online-1-console-init:1.4"
+                  value: "registry.redhat.io/amq7/amq-online-1-console-init:1.4-57"
                 - name: CONSOLE_SERVER_IMAGE
-                  value: "registry.redhat.io/amq7/amq-online-1-console-server-rhel7:1.4"
+                  value: "registry.redhat.io/amq7/amq-online-1-console-server-rhel7:1.4-28"
                 - name: CONSOLE_PROXY_OPENSHIFT_IMAGE
                   value: "registry.redhat.io/openshift3/oauth-proxy:latest"
                 - name: CONSOLE_PROXY_KUBERNETES_IMAGE
                   value: "quay.io/pusher/oauth2_proxy:latest"
                 - name: "FS_GROUP_FALLBACK_MAP"
                   value: "{}"
                 - name: ENABLE_MONITORING
-                  value: "true"
+                  value: "false"
```

Comment:
Is this something we always need to do for amq-online?

Reply:
Good question @mikenairn , No, it is just until https://issues.redhat.com/browse/INTLY-5111 is done

Reply:
OK, that's fine, i just want to make sure anything that can be automated is automated. Nothing to be done here if it's temporary.